app/pages/projects-disciplines.cjsx

```diff
@@ -1,15 +1,6 @@
-counterpart = require 'counterpart'
 React = require 'react'
 TitleMixin = require '../lib/title-mixin'
 apiClient = require 'panoptes-client/lib/api-client'
-OwnedCardList = require '../components/owned-card-list-by-discipline'
-
-counterpart.registerTranslations 'en',
```

Comment:
this set of messages already exists in `projects.cjsx` and in fact that is the one used, this one is unused so can be deleted.